### PR TITLE
Python 2/3 compatibility

### PR DESCRIPTION
**Pull request by arepa8**
This PR will solve the Python 2/3 compatibility

Tests results below:
```
-bitso-MacBook-Pro-de-Andres:bitso-py andreshernandez$ python -m test
.............................
----------------------------------------------------------------------
Ran 29 tests in 0.044s

OK
-bitso-MacBook-Pro-de-Andres:bitso-py andreshernandez$ python3 -m test
.............................
----------------------------------------------------------------------
Ran 29 tests in 0.042s

OK
```

### CODE REVIEW

**Comment by alejandro-martin** [comment]
In general, it looks good, just a couple of comments

File: bitso/api.py [hunk 1]
```diff
@@ -915,7 +930,11 @@ def _request_url(self, url, verb, params=None, private=False):
                 resp = requests.delete(url, headers=headers)
             except requests.RequestException as e:
                 raise
-        data = self._parse_json(resp.content.decode('utf-8'))
+        content = resp.content
+        data = self._parse_json(content
```
Comment (alejandro-martin) [inline bitso/api.py:934]: Could you also write this as an inline function?

File: bitso/api.py [hunk 2]
```diff
@@ -895,6 +906,10 @@ def _request_url(self, url, verb, params=None, private=False):
         headers=None
         if params == None:
             params = {}
+        params = {k:
```
Comment (alejandro-martin) [inline bitso/api.py:909]: Thanks for your contribution.
Could you use this way -> `k: v.decode("utf-8") if isinstance(v, bytes) else v for k, v in params.items()`
please.

**Comment by alejandro-martin** [comment]
It looks good, just a couple of comments.

File: bitso/api.py (hunk 2 above)
Comment (arepa8) [reply bitso/api.py]: Sure!
Thanks for your feedback.

File: bitso/api.py (hunk 1 above)
Comment (arepa8) [reply bitso/api.py]: Sure!

**Pull request merged**